postprocessing/janus-pp-rec.c

```diff
@@ -67,6 +67,10 @@ Usage: janus-pp-rec [OPTIONS] source.mjr [destination.[opus|wav|webm|mp4|srt]]
                                   7=maximum debug level; default=4)
   -D, --debug-timestamps        Enable debug/logging timestamps  (default=off)
   -o, --disable-colors          Disable color in the logging  (default=off)
+	-f, --format=STRING           Specifies the output format (overrides the
```

Comment:
The indentation here doesn't match the other commands, and looks like the description is incomplete too. Besides, not sure why there are three empty lines below?

Reply:
done.  description is continued in the following three lines.  Might be more obvious now I have corrected the whitespace formatting.